test/Microsoft.AspNetCore.WebSockets.ConformanceTest/Autobahn/AutobahnTester.cs

```diff
@@ -162,7 +162,9 @@ public void Verify(AutobahnResult result)
 
             var handler = new HttpClientHandler();
             // Win7 HttpClient on NetCoreApp2.1 defaults to TLS 1.0 and won't connect to Kestrel. https://github.com/dotnet/corefx/issues/28733
-            handler.SslProtocols = SslProtocols.Tls12 | SslProtocols.Tls11;
+            // Mac HttpClient on NetCoreApp2.0 doesn't alow you to set some combinations.
+            // https://github.com/dotnet/corefx/blob/586cffcdfdf23ad6c193a4bf37fce88a1bf69508/src/System.Net.Http/src/System/Net/Http/CurlHandler/CurlHandler.SslProvider.OSX.cs#L104-L106
+            handler.SslProtocols = SslProtocols.Tls12 | SslProtocols.Tls11 | SslProtocols.Tls;
```

Comment:
The previous code should work on mac in 2.1, though; are you seeing otherwise?  In 2.1 we're using SslStream, which uses Apple's libraries directly, and they support multiple protocols as long as they're contiguous, which the previous settings were.
cc: @bartonjs

Reply:
2.1 also failed but it was still using libcurl. https://travis-ci.org/aspnet/WebSockets/jobs/361385111

Reply:
> 2.1 also failed but it was still using libcurl

Ok.  Once it moves to a newer 2.1 that's not, the previous code should work, but obviously what you've changed it to is fine as well.